src/api.js

```diff
@@ -0,0 +1,12 @@
+const BASE_URL = 'http://www.omdbapi.com/?apikey=dd1635ad&';
```

Comment:
```suggestion
const BASE_URL = 'https://www.omdbapi.com/?apikey=dd1635ad&';
```
If you will check your page on the github by your demolink, you sill see that it's not working. 
Chaknge to `https`